mailbox.go

```diff
@@ -170,79 +163,5 @@ func (box *Mailbox) put(req *request) error {
 // Using a mailbox requires that the process creating the mailbox also
 // started a grid Server.
 func NewMailbox(s *Server, name string, size int) (*Mailbox, error) {
```

Comment:
This is deprecated now, right?

```suggestion
// started a grid Server.
//
// Deprecated: Use s.NewMailbox() instead.
func NewMailbox(s *Server, name string, size int) (*Mailbox, error) {
```